src/components/SQForm/SQFormAutocomplete.tsx

```diff
@@ -12,6 +12,7 @@ import {getIn, useField, useFormikContext} from 'formik';
 import {usePrevious} from '@selectquotelabs/sqhooks';
 import {useForm} from './useForm';
 import type {AutocompleteChangeReason} from '@material-ui/lab';
+import type {TextFieldProps} from '@material-ui/core';
```

Comment:
Since we are getting to the end of the MUI5 migration, are we ok with importing stuff from `material-ui/core` and backporting to our issue-447 branch or do we want to wait on this PR until the MUI5 migration is done and we can import `TextFieldProps` from MUIv5?

Reply:
Yes I will work on that.

Reply:
I have imported TextFieldProps from MUIv5 and resolved the conflicts.